pkg/scheduler/controller/common/seed.go

```diff
@@ -27,5 +27,12 @@ func VerifySeedReadiness(seed *gardencorev1beta1.Seed) bool {
 	if cond := gardencorev1beta1helper.GetCondition(seed.Status.Conditions, gardencorev1beta1.SeedGardenletReady); cond == nil || cond.Status != gardencorev1beta1.ConditionTrue {
 		return false
 	}
+
+	// Only consider condition if it's found because we haven't maintained it in previous releases.
+	// TODO: Handle this condition more conservatively in the future.
+	if cond := gardencorev1beta1helper.GetCondition(seed.Status.Conditions, gardencorev1beta1.SeedBackupBucketReady); cond != nil && cond.Status != gardencorev1beta1.ConditionTrue {
```

Comment:
Actually this condition is optional and should not be checked if `seed.spec.backup==nil`, right?

Reply:
Yes, that is also one of the reasons the condition is `cond != nil && cond.Status != gardencorev1beta1.ConditionTrue`.

Reply:
I added it in addition to make it clear.